src/main/java/frc/robot/subsystems/Elevator/Elevator.java

```diff
@@ -0,0 +1,123 @@
+package frc.robot.subsystems.elevator;
+
+import static edu.wpi.first.units.Units.Meters;
+import static edu.wpi.first.units.Units.Rotation;
+import static edu.wpi.first.units.Units.Rotations;
+import org.littletonrobotics.junction.Logger;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.units.measure.Distance;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+/**
+ * Elevator Subsystem
+ */
+public class Elevator extends SubsystemBase {
+    ElevatorIO io;
+    private ElevatorInputsAutoLogged inputs = new ElevatorInputsAutoLogged();
+
+    public Elevator(ElevatorIO io) {
+        this.io = io;
+        io.updateInputs(inputs);
+    }
+
+    @Override
+    public void periodic() {
+        io.updateInputs(inputs);
+        Logger.processInputs("Elevator", inputs);
+    }
+
+    /**
+     * converts angles to distance
+     *
+     * @param angle the angle the motor moves
+     * @return distance
+     */
+    private Distance angleToDistance(Angle angle) {
+        return Meters.of(angle.in(Rotation) * Constants.Elevator.gearRatio);
+    }
+
+    /**
+     * converts distance to angle
+     *
+     * @param distance the height we want converted
+     *
+     * @return angle
+     * 
+     */
+    private Angle distanceToAngle(Distance distance) {
+        return Rotations.of(distance.in(Meters) / Constants.Elevator.gearRatio);
+    }
+
+    /**
+     * moves elevator to home
+     *
+     * @return elevator at home
+     * 
+     */
+    public Command home() {
+        return raise(Constants.Elevator.HOME).andThen(() -> io.setVoltage(-0.1))
```

Comment:
`() -> io.setVoltage(-0.1)` is not a command and doesn't have an end action so the voltage won't stop